src/api/useAuth.jsx

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from 'react';
 import { auth } from './config.js';
-import { GoogleAuthProvider, signInWithRedirect } from 'firebase/auth';
+import { GoogleAuthProvider, signInWithPopup } from 'firebase/auth';
```

Comment:
I think these changes should not be included in this PR

Reply:
New commit is setting back to signInWithRedirect